lib/fastlane/actions/get_github_release.rb

```diff
@@ -109,6 +110,15 @@ def self.available_options
                                          raise "Please only pass the path, e.g. 'KrauseFx/fastlane'".red if value.include? "github.com"
                                          raise "Please only pass the path, e.g. 'KrauseFx/fastlane'".red if value.split('/').count != 2
                                        end),
+          FastlaneCore::ConfigItem.new(key: :server_url,
+                                       env_name: "FL_GITHUB_RELEASE_SERVER_URL",
+                                       description: "The server url. e.g. 'https://your.github.server/api/v3' (Default: 'https://api.github.com')",
+                                       default_value: "https://api.github.com",
+                                       optional: true,
+                                       verify_block: proc do |value|
+                                         raise "Please don't include the trailing slash in the server url".red if value.end_with? "/"
```

Comment:
Thank you for this PR, this is a nice addition!  I made a suggestion for normalizing the form it from the params.